src/SoundPlayer.js

```diff
@@ -1,13 +1,16 @@
 var Tone = require('tone');
 var log = require('./log');
 
-function SoundPlayer (outputNode) {
-    this.outputNode = outputNode;
+function SoundPlayer () {
+    this.outputNode;
     this.buffer; // a Tone.Buffer
     this.bufferSource;
```

Comment:
I'd suggest initializing these to safe default values, like `null`. You can also specify the type through a JSDoc comment like this example pulled from `Sprite` in the VM:
```js
    /**
     * Human-readable name for this sprite (and all clones).
     * @type {string}
     */
    this.name = '';
```